nautobot/ipam/api/views.py

```diff
@@ -140,7 +140,9 @@ def available_prefixes(self, request, pk=None):
         prefix = get_object_or_404(self.queryset, pk=pk)
         if request.method == "POST":
 
-            with cache.lock("available-prefixes", blocking_timeout=5):
+            with cache.lock(
+                "available-prefixes", blocking_timeout=5, timeout=get_settings_or_config("AVAILABLE_PREFIX_TIMEOUT")
+            ):
```

Comment:
Is there a similar issue with the "available-ips" feature?

Reply:
Yes there is, didn't think about that. I will update the var name and should along with comments.